diff -uNr nextcloud_orig/reactive/apache.py nextcloud/reactive/apache.py
--- nextcloud_orig/reactive/apache.py	2019-02-07 15:29:02.112829947 -0200
+++ nextcloud/reactive/apache.py	2019-02-07 15:33:13.615706901 -0200
@@ -94,6 +94,11 @@
     subdir = os.path.join(site_dir, contents[0])
     if not os.path.isdir(subdir):
         return
+    # check for the case where subdir have the same name of the parent
+    if os.path.basename(site_dir) == contents[0]:
+        os.rename(subdir, subdir+"_tmp")
+        contents = os.listdir(site_dir)
+        subdir = os.path.join(site_dir, contents[0])
     tmp_dest = os.path.join(os.path.dirname(site_dir), contents[0])
     os.rename(subdir, tmp_dest)
     os.rmdir(site_dir)
