diff -uNr nextcloud_orig/lib/charms/layer/basic.py nextcloud/lib/charms/layer/basic.py
--- nextcloud_orig/lib/charms/layer/basic.py	2019-02-07 15:28:57.204469437 -0200
+++ nextcloud/lib/charms/layer/basic.py	2019-02-07 15:30:13.997765849 -0200
@@ -115,8 +115,8 @@
         # re-enable installation from pypi
         os.remove('/root/.pydistutils.cfg')
         # install python packages from layer options
-        if cfg['python_packages']:
-            check_call([pip, 'install', '-U'] + cfg['python_packages'])
+        if cfg.get('python_packages'):
+            check_call([pip, 'install', '-U'] + cfg.get('python_packages'))
         if not cfg.get('use_venv'):
             # restore system pip to prevent `pip3 install -U pip`
             # from changing it
